### DISCUSSION TITLE
The built-in driver "SQL Server" and datetime data type

### DISCUSSION THREAD

**Discussion by puatrap** [mkleehammer/pyodbc#1210] [Q&A] [answered]
Please first make sure you have looked at:

* Documentation: https://github.com/mkleehammer/pyodbc/wiki
* Other issues

### Environment

To diagnose, we usually need to know the following, including version numbers.  On Windows, be
sure to specify 32-bit Python or 64-bit:

- Python: 3.11 64-bit
- pyodbc: 4.0.39
- OS: Windows 10
- DB: SQL Server
- driver: SQL Server

### Issue

Why is it that my new computer has no drivers installed except for the built-in "SQL Server," yet I am still able to retrieve datetime data types using the following code:
pyodbc.connect('DRIVER={SQL Server};SERVER=192.168.0.1;......'
Isn't it true that the built-in "SQL Server" does not support this data type?

My question is, when using mssql-django in Django, which also uses the built-in "SQL Server" as its driver, it gives me the error message "The database driver doesn't support modern datatime types."
Isn't mssql-django also developed based on pyodbc?
Why can't it retrieve data properly?

**Comment by gordthompson** [comment, marked as answer]
Cross-posted in a more appropriate location here:

https://github.com/microsoft/mssql-django/issues/262
